circle.yml

```diff
@@ -184,7 +184,71 @@ jobs:
       - store_artifacts:
           path: /tmp/xcode_raw.log
 
-  # Hockey
+  master:
```

Comment:
What's the `master` job for?

Reply:
Oh I see now - your intention is that every time we push to `master` it should also release an alpha build 🤔

Ok, we can try it this way - there may be another way to inject the environment variable though if that doesn't work.